### PR TITLE
Enable changelogs for collection

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

**Review by alinabuzachis — APPROVED** [review]

**Review by ssbarnea — CHANGES_REQUESTED** [review]
While I find it useful, I am not seeing this being used by any GHA jobs, so I have no confidence that these settings are working.

Please ensure that we have a way to test them on CI.

**Comment by Akasurde** [comment]
I think these are standard changelog settings. These will be used at the time of release. I tried them locally and they work.

**Review by ssbarnea — CHANGES_REQUESTED** [review]
I started to work on it but I faced some problems like https://github.com/ansible-community/antsibull-changelog/issues/172 
and I had to stop.

We should not merge this unless we can validate that we can build/test these fragments.

File: changelogs/.plugin-cache.yaml
Comment (felixfontein) [inline changelogs/.plugin-cache.yaml:1]: Don't check this file in. Add it to `.gitignore` instead.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -101,6 +101,11 @@ repos:
           - types-aiobotocore
           - watchdog
           - xxhash
+  - repo: https://github.com/ansible-community/antsibull-changelog
+    rev: 0.29.0
+    hooks:
+    - id: antsibull-changelog-lint
+    - id: antsibull-changelog-lint-changelog-yaml
```
Comment (felixfontein) [inline .pre-commit-config.yaml:108]: If you lint `changelogs/changelog.yaml`, you also need to add that file. Otherwise linting it will fail.

**Pull request merged**